listing-rewards/program/src/listings/create/mod.rs

```diff
@@ -69,12 +70,11 @@ pub struct CreateListing<'info> {
     pub wallet: Signer<'info>,
 
     /// SPL token account containing token for sale.
-    #[
-        account(
-            mut,
-            constraint = token_account.owner == wallet.key(),
-            constraint = token_account.amount == 1
-        )]
+    #[account(
+        mut,
+        constraint = token_account.owner == wallet.key(),
+        constraint = token_account.amount == 1
```

Comment:
Need to think about selling SFTs where the token amount is more than 1.

Reply:
Would greater equals work here? @kespinola